src/rd.h

```diff
@@ -79,13 +79,17 @@
 */
 static RD_INLINE RD_UNUSED void *rd_calloc(size_t num, size_t sz) {
 	void *p = calloc(num, sz);
+#ifndef _AIX  // On AIX, calloc(0) is NULL
```

Comment:
This assertion fails when running tests and I printed to check the size which was 0. 
 I didn't  witness any crash after adding this change.  As far as free() is concerned, it should not cause any issue because on IBM free(NULL) is a No-Op.